jgt_tools/utils.py

```diff
@@ -19,16 +21,16 @@ def execute_command_list(commands_to_run, verbose=True):
         subprocess.run(shlex.split(command), check=True)
 
 
-_DEFAULT_CONFIGS = {
-    "env_setup_commands": [
-        "pip install --upgrade pip<19",
-        "poetry install",
-        "pre-commit install",
-    ],
-    "self_check_commands": ["pre-commit run -a"],
-    "run_tests_commands": [f"{sys.executable} -m pytest"],
-    "doc_build_types": ["api"],
-}
+_DEFAULT_CONFIGS: defaultdict = defaultdict(list)
+
+
+def _load_defaults():
+    with open("data/defaults.csv") as f:
+        for group, cmd in csv.reader(f):
+            _DEFAULT_CONFIGS[group].append(cmd.replace("python", sys.executable))
```

Comment:
If the above suggestion about using `poetry` consistently is used, there's no need for any replacement here.